modin/pandas/test/test_series.py

```diff
@@ -4711,3 +4711,25 @@ def func(val):
     modin_series = modin_df["col"].apply(func)
 
     df_equals(modin_series, pandas_series)
+
+
+def test_apply_return_df():
+    modin_series, pandas_series = create_test_series([2, 4, 6, 0, 1])
+    eval_general(
+        modin_series,
+        pandas_series,
+        lambda series: series.apply(lambda x: pandas.Series([x, x + 1])),
+    )
+
```

Comment:
```suggestion
```

I don't see the need to test a small series in addition to a large series.

Reply:
I figured it would be better to test it with one partition and then with multiple partitions?